pkgs/secp256k1/default.nix

```diff
@@ -21,6 +21,6 @@ stdenv.mkDerivation {
     homepage = "https://github.com/bitcoin-core/secp256k1";
     license = with licenses; [ mit ];
     maintainers = with maintainers; [ nixbitcoin ];
-    platforms = with platforms; unix;
+    platforms = platforms.unix;
```

Comment:
Shouldn't the above lines also be edited in the same way?

Reply:
Fixed for `license`, but keep the list for `maintainers`, which is a nixpkgs convention.